### DISCUSSION TITLE
Adding wireguard support

### DISCUSSION THREAD

**Discussion by cbijon** [HASwitchPlate/openHASP#573] [Ideas]
Hello,

I would like to have the possibility to configure a VPN to use a remote homeassistant. Wireguard is available on esphome and i think it can be a good idea to have it there.
Someone can help me ?

**Comment by elik745i** [comment]
Im using keendns on my keenetics router, HA is assessible from outside even behind gray IP. Setup port forwarding and openhasp device conects to HA with no probs. If you do have static IP you can use any router.

**Comment by fvanroie** [comment]
This feature got added in #585 by @perexg but I'm unable to test this myself.
Please try the latest RC9 build from the Actions tab or from https://nightly.openhasp.com
